z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-install-config/tasks/main.yaml

```diff
@@ -39,7 +39,23 @@
       sed -i '/httpsProxy/a \ \ noProxy: {{ no_proxy }}' install-config.yaml
   when:
   - use_proxy == true 
-    
+
+- name: Add local registry info in install-config.yaml
+  shell:
+    cmd: |
+      sed -i '/baseDomain/a imageContentSources:' install-config.yaml
+      sed -i '/imageContentSources/a - mirrors:\n\ \ - {{ local_mirror_registry }}\n\ \ source: {{ local_mirror_source_release }} ' install-config.yaml
+      sed -i '/imageContentSources/a - mirrors:\n\ \ - {{ local_mirror_registry }}\n\ \ source: {{ local_mirror_source_dev }} ' install-config.yaml
+  when:
+  - disconnect_installation == true
+
+- name: Add additional certificates to install-config.yaml
+  lineinfile:
+    line: '{{ additional_certs }}'
+    path: install-config.yaml
+    insertafter: EOF
+  when: additional_certs is defined
```

Comment:
Is additional_certs is defined too in connected case?

Reply:
No, connected case doesn't need `additional_certs`.